pkg/operation/botanist/controlplane.go

```diff
@@ -1079,52 +1084,99 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 		"storageCapacity": b.Seed.GetValidVolumeSize("10Gi"),
 	}
 
-	etcd, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	imageNames := []string{
+		common.ETCDImageName,
+		common.ETCDBackupRestoreImageName,
+	}
+
+	values, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	if err != nil {
+		return err
+	}
+
+	images, err := imagevector.FindImages(b.ImageVector, imageNames, imagevector.RuntimeVersion(b.SeedVersion()), imagevector.TargetVersion(b.ShootVersion()))
 	if err != nil {
 		return err
 	}
 
+	values["prefix"] = common.GenerateBackupEntryName(b.Shoot.Info.Status.TechnicalID, b.Shoot.Info.Status.UID)
+
 	for _, role := range []string{common.EtcdRoleMain, common.EtcdRoleEvents} {
-		etcd["role"] = role
+		values["global"] = map[string]interface{}{
+			"images": chart.ImageMapToValues(images),
+		}
+		values["role"] = role
+		values["backupEnabled"] = false
+		values["provider"] = ""
 		if role == common.EtcdRoleMain {
 			// etcd-main emits extensive (histogram) metrics
-			etcd["metrics"] = "extensive"
+			values["metrics"] = "extensive"
+			if b.Seed.Info.Spec.Backup != nil {
+				values["provider"] = b.Seed.Info.Spec.Backup.Provider
+				values["backupEnabled"] = true
+				secret := &corev1.Secret{}
+				if err := b.K8sSeedClient.Client().Get(ctx, kutil.Key(b.Shoot.SeedNamespace, "etcd-backup"), secret); err != nil {
```

Comment:
Use the already defined constant for `etcd-backup`.

Reply:
Done